code/backend/batch/utilities/helpers/AzureBlobStorageHelper.py

```diff
@@ -7,10 +7,34 @@
     ContentSettings,
     UserDelegationKey,
 )
+from azure.storage.queue import QueueClient, BinaryBase64EncodePolicy
 from .EnvHelper import EnvHelper
 from azure.identity import DefaultAzureCredential
 
 
+def connection_string(account_name: str = None, account_key: str = None):
```

Comment:
Maybe it's better to not have defaults here because we wouldn't want the connection string to contain `None`, what do you think?

Reply:
Oh yeah that's true, I think my idea was to get them from env helper here but actually I did it outside of the method instead. Will change